### PR TITLE
refs platform/1307: hide drupal headers by default

### PR DESCRIPTION
**Pull request by Monska85**

### CODE REVIEW

**Comment by paolomainardi** [comment]
Merging this will make all the drupal apps also for development branches not printing these headers anymore, we can fix it by releasing a new package enabling but most of the projects will be affected.
Let's dig it more

**Review by paolomainardi — CHANGES_REQUESTED** [review]
We should understand better the effects on local/dev envs and how to handle them

**Comment by Monska85** [comment]
I have opened a MR to restore the headers in the package for the local and the review environments.

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -48,9 +48,9 @@ export NGINX_CACHE_CONTROL_HEADER="${NGINX_CACHE_CONTROL_HEADER:-public,max-age=
 export NGINX_GZIP_ENABLE="${NGINX_GZIP_ENABLE:-1}"
 export SITEMAP_URL="${SITEMAP_URL}"
 export NGINX_REDIRECT_FROM_TO_WWW="${NGINX_REDIRECT_FROM_TO_WWW:-0}"
-export NGINX_HIDE_DRUPAL_HEADERS="${NGINX_HIDE_DRUPAL_HEADERS:-0}"
+export NGINX_HIDE_DRUPAL_HEADERS="${NGINX_HIDE_DRUPAL_HEADERS:-1}"
 export NGINX_HIDE_SENSITIVE_HEADERS="${NGINX_HIDE_SENSITIVE_HEADERS:-1}"
-export NGINX_XFRAME_OPTION_ENABLE="${NGINX_XFRAME_OPTION_ENABLE:-0}"
+export NGINX_XFRAME_OPTION_ENABLE="${NGINX_XFRAME_OPTION_ENABLE:-1}"
```
Comment (Syphon83) [inline docker-entrypoint.sh]: We should leave X-frame options header disabled by default, do you agree?

**Review by Syphon83 — APPROVED** [review]
LGTM ! Thank you !

File: docker-entrypoint.sh (hunk 1 above)
Comment (Monska85) [reply docker-entrypoint.sh]: Disabled

**Pull request merged**